README.md

```diff
@@ -1,8 +1,74 @@
-# api-linter-service
+# API Linting Service
 
-[![SIT](https://img.shields.io/badge/SIT-awesome-blueviolet.svg)](https://jobs.schwarz)
```

Comment:
You don't want to keep the SIT badge?

Reply:
done

Reply:
I think he is removing it so I can do my traditional "add the SIT Badge" commit :)